test/unit/api/cpp/solver_black.cpp

```diff
@@ -190,6 +190,20 @@ TEST_F(TestApiBlackSolver, mkFiniteFieldSort)
 {
   ASSERT_NO_THROW(d_solver.mkFiniteFieldSort("31"));
   ASSERT_THROW(d_solver.mkFiniteFieldSort("6"), CVC5ApiException);
+
+  ASSERT_NO_THROW(d_solver.mkFiniteFieldSort("1100101",2));
+  ASSERT_NO_THROW(d_solver.mkFiniteFieldSort("10202", 3));
+  ASSERT_NO_THROW(d_solver.mkFiniteFieldSort("401",   5));
+  ASSERT_NO_THROW(d_solver.mkFiniteFieldSort("791a", 11));
+  ASSERT_NO_THROW(d_solver.mkFiniteFieldSort("970f", 16));
+  ASSERT_NO_THROW(d_solver.mkFiniteFieldSort("8CC5", 16));
+  
+  ASSERT_THROW(d_solver.mkFiniteFieldSort("1100100",2), CVC5ApiException);
+  ASSERT_THROW(d_solver.mkFiniteFieldSort("10201", 3), CVC5ApiException);
+  ASSERT_THROW(d_solver.mkFiniteFieldSort("400",   5),CVC5ApiException);
+  ASSERT_THROW(d_solver.mkFiniteFieldSort("7919", 11),CVC5ApiException);
+  ASSERT_THROW(d_solver.mkFiniteFieldSort("970e", 16),CVC5ApiException);
+  ASSERT_THROW(d_solver.mkFiniteFieldSort("8CC4", 16),CVC5ApiException);
```

Comment:
Same as with `mkFiniteFieldElem` below, @Sarkoxed can you add these and the existing ones to a new test in `test/unit/api/python/test_solver.py`?